gulpfile.js

```diff
@@ -155,7 +162,7 @@ gulp.task('doc', function() {
 
 // Generated Sassdoc to Dist folder
 gulp.task('doc:dist', function() {
-  return gulp.src(config.folderAssets.base + '/**/*.scss')
+  return gulp.src('/' + config.folderAssets.base + '/**/*.scss')
```

Comment:
How does it affect the result?

Reply:
This is not supposed to be part of the change, I will remove it, thanks!